civihr_default_theme/scss/components/_leave-request-modal.scss

```diff
@@ -55,3 +55,16 @@ $row-margin-bottom: 15px;
     margin-top: 18px;
   }
 }
+
+.chr_leave-request-modal__comments-container {
```

Comment:
See comment i left in https://github.com/civicrm/civihr/pull/1483 , if you do that improvement, you're gonna have a separate file for the leave request component itself